### DISCUSSION TITLE
NZXT Zxx LCD gpu/cpu temperature and color support

### DISCUSSION THREAD

**Discussion by arolls** [liquidctl/liquidctl#562] [Ideas]
**New Feature Request**

- It would be really useful to have support for both cpu and gpu temperatures on the LCD screen of the KZXT Kraken Z series watercoolers. I'm sure this is not a huge quantum leap since liquid temperature has been implemented in 1.11.x.
- In addition to the above, having the ability to set the colour of the text on the screen would be an added bonus. Current the lcd display only displays white text.

**Comment by jonasmalacofilho** [comment]
Actually both features would indeed be quite significant leaps: they would require liquidctl to keep running in the background to send periodic updates to the cooler:

- the device has no idea about, and no way to access by itself, any sensors other than its own liquid temperature probe;
- the liquid temperature display mode is a firmware feature, and AFAIK it isn't customizable; the only way to do it would be to reimplement it in software (and thus require constant updates).

**Reply by Avnsx** [reply]
Did I understand correctly, you don't want to implement CPU & GPU temperature reading & display on the LCD screen, because it would mean that liquidctl has to constantly poll temperatures & and then update them on the LCD screen?

Regardless of polling or not, I would be down to maybe spend like 0.5% cpu utilisation every 3 seconds to do the above described task. I mean NZXT CAM itself, was unironically managing to spike my GPU usage to 80% when it was just in system tray ..

**Reply by markmann447** [reply]
I would also like to ask about this since I would like for my LCD to monitor my CPU temps as well.
